apps/admin-x-activitypub/src/components/feed/FeedItemMenu.tsx

```diff
@@ -0,0 +1,90 @@
+import {
+    AlertDialog,
+    AlertDialogAction,
+    AlertDialogCancel,
+    AlertDialogContent,
+    AlertDialogDescription,
+    AlertDialogFooter,
+    AlertDialogHeader,
+    AlertDialogTitle,
+    AlertDialogTrigger,
+    Button,
+    Popover,
+    PopoverClose,
+    PopoverContent,
+    PopoverTrigger,
+    buttonVariants
+} from '@tryghost/shade';
+
+interface FeedItemMenuProps {
+    trigger: React.ReactNode;
```

Comment:
I really like this pattern of passing the trigger to the dialog! What do you think about instead of passing on a ReactNode component as a prop, so the menu could be used like this:

```
<FeedItemMenu onCopyLink={...} onDelete={...}>
   <Button>Open menu</Button>
</FeedItemMenu>
```

Does this make sense to you?